<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Lukačišin Lab - Publications</title>
  <meta name="description" content="Lukačišin Lab -- Publications.">
  <link rel="stylesheet" href="https://lukacisinlab.github.io/css/main.css">
  <link rel="canonical" href="https://lukacisinlab.github.io/publications/">



</head>


  <body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container-fluid">
	<div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
	  </button>
	
    <a class="navbar-brand" href="https://lukacisinlab.github.io/">Lukačišin Lab &rarr; SysBio @ Comenius</a>
	</div>
	<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li><a href="https://lukacisinlab.github.io/">Home</a></li>
		<li><a href="https://lukacisinlab.github.io/team">Team</a></li>
		<li><a href="https://lukacisinlab.github.io/vacancies">Vacancies</a></li>
		<li><a href="https://lukacisinlab.github.io/publications">Publications</a></li>
		<li><a href="https://lukacisinlab.github.io/research">Research</a></li>
		<li><a href="https://lukacisinlab.github.io/pictures">(Pics)</a></li>
	  </ul>
	</div>
  </div>
</div>


    <div class="container-fluid">
      <div class="row">
        <div id="gridid" class="col-sm-12">
  <h1 id="publications">Publications</h1>

<h2 id="highlights">Highlights</h2>

<p><strong>At the end of this page, you can find the <a href="#full-list-of-publications">full list of publications</a>. All papers are also available on <a href="https://scholar.google.com/citations?user=LuEHJToAAAAJ">GoogleScholar</a>.</strong></p>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Interactions between immune cell types facilitate the evolution of immune traits</pubtit>
      <p><img src="https://lukacisinlab.github.io/images/pubpic/interaction.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Why is there so many subtypes of immune cells? In this work, we look at what allows the immune system to evolve quicker than other systems in mammalian organisms and we find that it is the modular organisation of the immune system into many interacting cell types that allows a quick evolution of the genes involved in the interaction while preserving the core function of each cell type.</p>
      <p><em>Dubovik*, T., Lukačišin*, M., Starosvetsky, E., LeRoy, B., Normand, R., Admon, Y., Alpert, A., Ofran, Y., G'Sell, M. &amp; Shen-Orr, S.S.</em></p>
      <p><strong><a href="https://www.nature.com/articles/s41586-024-07661-0">Nature 632:350–356 (2024)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Intron-mediated induction of phenotypic heterogeneity</pubtit>
      <p><img src="https://lukacisinlab.github.io/images/pubpic/yeast.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>What is the function of abundant introns, otherwise very rare in the budding yeast, in the ribosomal protein genes? Using isogrowth profiling to carefully control for growth rate while perturbing the cells with osmotic stress we discover a new function for introns - diversification of the isogenic population into two subgroups, each of those better equipped to different scenarios regarding nutrient abundance.</p>
      <p><em>Lukačišin*, M., Espinosa-Cantú*, A., Bollenbach, T.</em></p>
      <p><strong><a href="https://www.nature.com/articles/s41586-022-04633-0">Nature 605:113–118 (2022)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Emergent gene expression responses to drug combinations predict higher-order drug interactions</pubtit>
      <p><img src="https://lukacisinlab.github.io/images/pubpic/emergent.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Gene expression changes due to inhibition by small molecule inhibitors should in principle give us clues about mechanism-of-action of the drug and also allow us to predict its interaction with other drugs. However, the measurement of drug-specific effect are far too often confounded by gene expression changes due to altered growth rate. Here we develop isogrowth profiling, a method where we titre out the growth inhibition precisely using various ratios of two diffent inhibitors. This allowed us to disentangle the growht inhibition-dependent, drug-specific and drug combination-specific gene expression changes and predict higher-order drug antagonisms.</p>
      <p><em>Lukačišin, M., Bollenbach, T.</em></p>
      <p><strong><a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30349-7">Cell Systems 9:423-433 (2019)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Sequence-specific thermodynamic properties of nucleic acids influence both transcriptional pausing and backtracking in yeast</pubtit>
      <p><img src="https://lukacisinlab.github.io/images/pubpic/pausing.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>What makes RNA polymerase II pause during transcription? In eukaryotes, the prevailing opinion has been that it is the incompletely removed nucleosomes. In this work, we show that basepairing energies in front and behind the transcription bubble are sufficient to explain the pausing observed using genome-wide methods and do so much better than known nucleosomes positions. By deriving emprical relationship between basepairing energy difference and pausing propensity, we further go on to show that decreased basepairing of mismatched RNA:DNA is sufficient for triggering error correction by backtracking, widely believed to be dependent on kinetic discrimination.</p>
      <p><em>Lukačišin*, M., Landon*, M., Jajoo*, R.</em></p>
      <p><strong><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0174066">PLoS ONE 12(3): e0174066 (2017)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Elf5 inhibits the epithelial–mesenchymal transition in mammary gland development and breast cancer metastasis by transcriptionally repressing Snail2</pubtit>
      <p><img src="https://lukacisinlab.github.io/images/pubpic/emt.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Epithelial to mesenchymal transition is a fundamental process in both development and cancer progression. In this work we show that the transcription factor Elf5 acts not only in breast tissue development, but is also an upstream regulator of the key EMT inducer Snail2. This makes Elf5 the upstream-most known regulator of cell morphology changes required for tumour cell invasiveness and metastasis.</p>
      <p><em>Chakrabarti, R., Hwang, J., Blanco, M.A., Wei, Y., Lukačišin, M., Romano, R.A., Smalley, K., Liu, S., Yang, Q., Ibrahim, T., Mercatali, L., Amadori, D., Haffty, B.G., Sinha, S. and Kang, Y.</em></p>
      <p><strong><a href="https://www.nature.com/articles/ncb2607">Nature Cell Biology 14, 1212–1222 (2012)</a></strong></p>
      <p class="text-danger"><strong> <a style="color:#000000" href="https://www.nature.com/articles/ncb2616">Editorial by ​Mathsyaraja, H. and Ostrowski, M.C. Nat Cell Biol, 14:1122-3 (2012).</a></strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Rituximab induces sustained reduction of pathogenic memory B cell expansions during peripheral nervous system autoimmunity</pubtit>
      <p><img src="https://lukacisinlab.github.io/images/pubpic/jci.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>By mapping the repertoire of IgM-memory B cells in patients with anti-myelin associated glycoprotein neuropathy, we showed that B cell depletion by rituximab (anti-CD19) helps the responders by reducing clonal expansions of autoreactive B cells.</p>
      <p><em>Maurer, M.A., Rakocevic, G., Leung, C.S., Quast, I., Lukačišin, M., Goebels, N., Münz, C., Wardemann, H., Dalakas, M. and Lünemann, J.D.</em></p>
      <p><strong><a href="https://www.jci.org/articles/view/58743">Journal of Clinical Investigation 122:1393-402 (2012)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<p>   </p>

<h2 id="full-list-of-publications">Full List of publications</h2>

<p>Interactions between immune cell types facilitate the evolution of immune traits <br />
  <em>Dubovik*, T., Lukačišin*, M., Starosvetsky, E., LeRoy, B., Normand, R., Admon, Y., Alpert, A., Ofran, Y., G'Sell, M. &amp; Shen-Orr, S.S. </em><br /><a href="https://www.nature.com/articles/s41586-024-07661-0">Nature 632:350–356 (2024)</a></p>

<p>Single‐cell isogrowth profiling: Uniform inhibition uncovers non‐uniform drug responses <br />
  <em>Lukačišin, M., Espinosa-Cantú, A., Bollenbach, T. </em><br /><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ctm2.1005">Clinical and Translational Medicine 12(8):e1005 (2022)</a></p>

<p>Intron-mediated induction of phenotypic heterogeneity <br />
  <em>Lukačišin*, M., Espinosa-Cantú*, A., Bollenbach, T. </em><br /><a href="https://www.nature.com/articles/s41586-022-04633-0">Nature 605:113–118 (2022)</a></p>

<p>Emergent gene expression responses to drug combinations predict higher-order drug interactions <br />
  <em>Lukačišin, M., Bollenbach, T. </em><br /><a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30349-7">Cell Systems 9:423-433 (2019)</a></p>

<p>Sequence-specific thermodynamic properties of nucleic acids influence both transcriptional pausing and backtracking in yeast <br />
  <em>Lukačišin*, M., Landon*, M., Jajoo*, R. </em><br /><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0174066">PLoS ONE 12(3): e0174066 (2017)</a></p>

<p>Elf5 inhibits the epithelial–mesenchymal transition in mammary gland development and breast cancer metastasis by transcriptionally repressing Snail2 <br />
  <em>Chakrabarti, R., Hwang, J., Blanco, M.A., Wei, Y., Lukačišin, M., Romano, R.A., Smalley, K., Liu, S., Yang, Q., Ibrahim, T., Mercatali, L., Amadori, D., Haffty, B.G., Sinha, S. and Kang, Y. </em><br /><a href="https://www.nature.com/articles/ncb2607">Nature Cell Biology 14, 1212–1222 (2012)</a></p>

<p>Rituximab induces sustained reduction of pathogenic memory B cell expansions during peripheral nervous system autoimmunity <br />
  <em>Maurer, M.A., Rakocevic, G., Leung, C.S., Quast, I., Lukačišin, M., Goebels, N., Münz, C., Wardemann, H., Dalakas, M. and Lünemann, J.D. </em><br /><a href="https://www.jci.org/articles/view/58743">Journal of Clinical Investigation 122:1393-402 (2012)</a></p>


</div>

      </div>
    </div>

    <div id="footer" class="panel">
  <div class="panel-footer">
	<div class="container-fluid">
	  <div class="row">
		<div class="col-sm-4">

		  <p>&copy 2025 Lukačišin Lab. Site based on the <a href="https://www.allanlab.org/">Allan lab template</a>, made with <a href="https://jekyllrb.com">Jekyll</a>;  <a href="https://lukacisinlab.github.io/aboutwebsite.html">copy and  modify it for your own research group.</a></p>
              


		   <p>  </p><p>


		</div>
		<div class="col-sm-4">
		<p>We are part of the <a href="https://fns.uniba.sk/en/kge//">Deparmtent of Genetics</a> at the<a href="https://fns.uniba.sk/en/"> Faculty of Natural Sciences</a>, Comenius University, Bratislava, Slovakia.</p>
		</div>
		<div class="col-sm-4">
		  Contact:<br />
Ilkovičova 6<br />
		  842 15 Bratislava 4<br />
		  Slovakia <br />
		</div>
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="https://lukacisinlab.github.io/js/bootstrap.min.js"></script>


  </body>

</html>
